packages/api/endpoints/executions.js

```diff
@@ -2,9 +2,16 @@
 
 const router = require('express-promise-router')();
 const { RecordDoesNotExist } = require('@cumulus/errors');
-const { getKnexClient, ExecutionPgModel } = require('@cumulus/db');
+const {
+  getKnexClient,
+  ExecutionPgModel,
+  GranulesExecutionsPgModel,
+  GranulePgModel,
+} = require('@cumulus/db');
 const Search = require('@cumulus/es-client/search').Search;
 const models = require('../models');
+const { getGranuleIdsForPayload } = require('../lambdas/bulk-operation');
```

Comment:
Nit: If we need to re-use this helper here, I'd prefer to move it somewhere like `lib/` rather than requiring it from another lambda file

Reply:
Nitx2: Agree, and (and this is entirely not *this* PR, but given we're moving it) consider cleaning up the logic branches as well (e.g factor out  

```
  const uniqueGranuleIds = [...new Set(granuleIds)];
  return uniqueGranuleIds;
  
```
into a method and do something like (but less terse, ternary is for brevity):

 granule.ids.length ? uniqueify(granuleIds) : uniqueify(queryElasticSearhc(query)); in the parent method.

Not nit: we shouldn't be requiring cross /lambdas